app/assets/javascripts/application.js

```diff
@@ -16,4 +16,10 @@
 //= require turbolinks
 //= require_tree .
 
-$(function(){ $(document).foundation(); });
+$(function(){
+  $(document).foundation();
+  SC.initialize({
+    client_id: "b99714ff12392f9d7e1498ab5166732d",
```

Comment:
When you extract the above (heh-heh) to `initialize.js.erb`, you could pass this key in from `ENV`.  That would be sick yo.